apps/web/components/booking/DatePicker.tsx

```diff
@@ -259,7 +262,12 @@ function DatePicker({
         </div>
       </div>
       <div className="border-bookinglightest grid grid-cols-7 gap-4 border-t border-b text-center dark:border-gray-800 sm:border-0">
-        {weekdayNames(i18n.language, weekStart === "Sunday" ? 0 : 1, "short").map((weekDay) => (
+        {weekdayNames(
+          i18n.language,
+          // @note: Double ternary operator to support also Saturday () as start of week
+          weekStart === "Sunday" ? 0 : weekStart === "Monday" ? 1 : 6,
```

Comment:
Maybe we should make it more obvious and error-proof by making it very explicit in code instead of explaining in a comment

Reply:
Yeah I was thikning about moving to a function or something. We could probably code make support any -weekStart- too, thought about that to just proof-case any future need of -other- days as start of week.
Thoughts?

Reply:
@emrysal you're also working on the datepicker, so your input would be greatly appreciated on what you think it's best here 🙏🏻

Reply:
A function with early returns would be nice for legibility

Reply:
Thanks for the tag @agustif - I'm changing the way this works in the new DatePicker and it is already part of the Draft PR - this case is covered, as is Friday, Thursday, we can make any day of the week the first day of the week.